{"uuid":"81dfbc45-a2ef-42eb-b9f6-b2d3f2b74374","historyId":"73e0f93ac878201a48ea41728b2b1709","fullName":"src/test/resources/features/05_Array.feature:261","labels":[{"name":"tag","value":"Array"},{"name":"tag","value":"TS_array_31"},{"name":"host","value":"Anjus-Air"},{"name":"thread","value":"13037@Anjus-Air.main(1)"},{"name":"feature","value":"Array feature"},{"name":"story","value":"The user is presented with error message for code with invalid syntax in tryEditor for Find Numbers with Even Number of Digits"},{"name":"suite","value":"Array feature"},{"name":"testClass","value":"The user is presented with error message for code with invalid syntax in tryEditor for Find Numbers with Even Number of Digits"},{"name":"framework","value":"cucumber4jvm"},{"name":"language","value":"java"},{"name":"gherkin_uri","value":"file:///Users/anjujohn/eclipse-workspace5/DS_Algo_Project/src/test/resources/features/05_Array.feature"},{"name":"package","value":"src.test.resources.features.05_Array_feature.Array feature"}],"links":[],"name":"The user is presented with error message for code with invalid syntax in tryEditor for Find Numbers with Even Number of Digits","status":"passed","stage":"finished","steps":[{"name":"Given  The user is on Question page of \"Find Numbers with Even Number of Digits\" after logged in","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1704549535422,"stop":1704549535594},{"name":"When  The user enter python code with invalid syntax in tryEditor from sheet \"pythonCode\" and 1","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1704549535596,"stop":1704549535826},{"name":"And  The user clicks on run button","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1704549535827,"stop":1704549535943},{"name":"Then  The user should be presented with error message as \"SyntaxError: bad input on line 1\"","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1704549535945,"stop":1704549535956}],"attachments":[],"parameters":[{"name":"Sheetname","value":"pythonCode"},{"name":"RowNumber","value":"1"}],"start":1704549535420,"stop":1704549535957}